src/main/java/com/google/sps/servlets/URLTest.java

```diff
@@ -0,0 +1,185 @@
+      
+   // Create a Java URL to connect to the CSV we want to read from.
+      URL url = new URL("https://raw.githubusercontent.com/nytimes/covid-19-data/master/live/us-counties.csv");
+      
+   // From that URL, send a "Get" request to retrieve the information.
+      URLConnection data = url.openConnection();
+      
+   // Create a Scanner that can read from our URL response.
+      Scanner input = new Scanner(data.getInputStream());
+      
+      
+// #######################   Notes:
+// # Processing the Data #      Since I want the data to be indexed by State AND County, I must use "Pairs."
+// #######################      Also, since the data we have for every county is also a pair, I'll use one in our data aswell.
+
+   // Create a hashmap to store our data, indexed by a pair of strings, that will store a pair of ints. (State, County) => (cases, deaths)
+      HashMap<Pair<String, String>, Pair<Integer, Integer>> dataMap = new HashMap<Pair<String, String>, Pair<Integer, Integer>>();
+      
+   // Skip the first line of our file, it only contains table headers
+      input.nextLine();
+      
+   // Fill the hashmap with our data! (Read our data line by line, splitting by comma)
+      while(input.hasNextLine()) {
+         String[] tokens = input.nextLine().split(",");
+         
+         String county = tokens[1];
+         String state  = tokens[2];
+         
+         // DEBUG: System.out.println("State: " + state + " County: " + county + " Cases: " + tokens[4] + " Deaths: " + tokens[5]);
+         
+         int cases  = tokens[4].isEmpty() ? 0 : Integer.parseInt(tokens[4]);
+         int deaths = tokens[5].isEmpty() ? 0 : Integer.parseInt(tokens[5]);
+         
+         Pair<String, String> myCounty = new Pair<String, String>(state, county);
+         Pair<Integer, Integer> myCountyData = new Pair<Integer, Integer>(cases, deaths);
+         
+         dataMap.put(myCounty, myCountyData);
+      }
+      
+      input.close();
+
+
+// #############################
+// # Displaying Output To User #
+// #############################
+
+   // Create a Scanner that can read from the keyboard.
+      input = new Scanner(System.in);
+      
+   // Welcome the user to our prototype! And ask them to input the desired state and county variables.
+      System.out.println("Welcome to the Is Covid Near Me Java Prototype!\n");
+      System.out.println("Please enter the State and County you would like to see covid results for:");
+      
+      String state  = "";
+      String county = "";
+      Pair<String, String> myCounty = null;
+      
+   // Event loop to repeat if a user's desired state and county is not found, or an error happens during input.
+      while (true) {
+         System.out.print("State: ");
+         state = input.nextLine();
+         
+         System.out.print("County: ");
+         county = input.nextLine();
+         
+         myCounty = new Pair<String, String>(state, county);
+         
```

Comment:
I believe in tandem with changing the while loop changing the if-statement to follow would eliminate the need for a break statement:

if ( !dataMap.containsKey(myCounty) ) {
           System.out.println("Sorry! It looks like your county, " + county + " county in " + state + " is not in our list.");
           System.out.println("Please try again!\n");
}